poetry.lock

```diff
@@ -671,25 +671,25 @@ files = [
 
 [[package]]
 name = "requests"
-version = "2.28.1"
+version = "2.27.1"
```

Comment:
it seems requests has a very similar issue issue as urlib3 (urllib3/urllib3#2761) as of https://github.com/psf/requests/commit/8bce583b9547c7b82d44c8e97f37cf9a16cbe758 ([relevant diff](https://github.com/psf/requests/commit/8bce583b9547c7b82d44c8e97f37cf9a16cbe758#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L81-R96)).

I don't mind using this version of requests (for now) b/c I'm not aware of any CVEs, but I ought to open an issue on `requests` prior to merging (and open a PR to fix it).